Bug in action for PyPI release

```
Run python setup.py sdist bdist_wheel
  python setup.py sdist bdist_wheel
  python -m twine upload --skip-existing dist/*
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.10.12/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.10.12/x64/lib
    TWINE_USERNAME: __token__
    TWINE_PASSWORD: ***
/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py:76: _MissingDynamic: `license` defined outside of `pyproject.toml` is ignored.
!!
        ********************************************************************************
        The following seems to be defined outside of `pyproject.toml`:
        `license = 'MIT'`
        According to the spec (see the link below), however, setuptools CANNOT
        consider this value unless `license` is listed as `dynamic`.
        https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table
        To prevent this problem, you can list `license` under `dynamic` or alternatively
        remove the `[project]` table from your file and rely entirely on other means of
        configuration.
        ********************************************************************************
!!
  _handle_missing_dynamic(dist, project_table)
/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py:83: SetuptoolsWarning: `install_requires` overwritten in `pyproject.toml` (dependencies)
  corresp(dist, value, root_dir)
Traceback (most recent call last):
  File "/home/runner/work/sparkx/sparkx/setup.py", line 3, in <module>
    setup(
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/__init__.py", line 104, in setup
    return distutils.core.setup(**attrs)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 158, in setup
    dist.parse_config_files()
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/dist.py", line 631, in parse_config_files
    pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 69, in apply_configuration
    return _apply(dist, config, filepath)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 57, in apply
    _apply_project_table(dist, config, root_dir)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 83, in _apply_project_table
    corresp(dist, value, root_dir)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 184, in _license
    _set_config(dist, "license", val["text"])
KeyError: 'text'
Error: Process completed with exit code 1.
```

Here is the error message:
```
Run python setup.py sdist bdist_wheel
/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/setuptools/config/_apply_pyprojecttoml.py:83: SetuptoolsWarning: `install_requires` overwritten in `pyproject.toml` (dependencies)
  corresp(dist, value, root_dir)
running sdist
running egg_info
error: error in 'egg_base' option: 'sparkx' does not exist or is not a directory
Error: Process completed with exit code 1.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "sparkx"
@@ -37,5 +37,3 @@ dependencies = [
 "Repository" = "https://github.com/smash-transport/sparkx"
 "Bug Tracker" = "https://github.com/smash-transport/sparkx/issues"
 "Documentation" = "https://smash-transport.github.io/sparkx/"
-
-

--- a/setup.py
+++ b/setup.py
@@ -2,8 +2,8 @@ from setuptools import find_packages, setup
 
 setup(
     name='sparkx',
-    packages=find_packages(where='src/sparkx'),
-    package_dir={'': 'sparkx'},
+    packages=find_packages(where='src'),
+    package_dir={'': 'src'},
     install_requires=[
         "particle==0.23.0",
         "numpy>=1.23.5",

